openapi.json

```diff
@@ -1509,7 +1509,7 @@
           },
           {
             "lang": "javascript--nodejs",
-            "source": "const sstk = require('shutterstock-api');\r\n\r\nsstk.setAccessToken(process.env.SHUTTERSTOCK_API_TOKEN);\r\n\r\nconst imagesApi = new sstk.ImagesApi();\r\n\r\nconst body = {\r\n  \"name\": \"My collection\"\r\n};\r\n\r\nimagesApi.createLightbox(body)\r\n.then(function() {\r\n  console.log(\"Success!\");\r\n})\r\n.catch(function(error) {\r\n  console.error(error);\r\n});\r\n"
+            "source": "const sstk = require('shutterstock-api');\r\n\r\nsstk.setAccessToken(process.env.SHUTTERSTOCK_API_TOKEN);\r\n\r\nconst imagesApi = new sstk.ImagesApi();\r\n\r\nconst body = {\r\n  \"name\": \"My collection\"\r\n};\r\n\r\nimagesApi.createLightbox(body)\r\n  .catch((error) => {\r\n    console.error(error);\r\n  });\r\n"
```

Comment:
what happened to the `.then` block here?

Reply:
I took the `then` block out of methods that don't return any data. Should I put it back in and write "Done!" to the console or something like that?

Reply:
I think that may make sense. @anthonyjonahs what do you think?

Reply:
I think it's fine for a `POST` request like this.